### PR TITLE
#128 Add custom initialization at start time (1.4.0)

### PR DESCRIPTION
**Pull request by carlosmonteagudoinbibo**
### Changes
- Update attribute tables and AET screenshots in all deformers to document the attributes for custom constraints initialization at start time

#### Skin

image

image

#### Fat

image

image

#### Muscle

image

image

#### Ribbon

image

image

#### Simshape

I decided to expose the content in the order that it should appear. We will fix the AET for AdnSimshape in the next patch/release

image

image

### CODE REVIEW

**Review by josevalverdeinbibo — APPROVED** [review]
Approved with a comment.

File: docs/maya/fat.md [hunk 1]
```diff
@@ -56,6 +56,12 @@ The process to create an AdnFat deformer is:
 
 ### Advanced Settings
 
+#### Initialization Settings
+| Name | Type | Default | Animatable | Description |
+| :--- | :--- | :------ | :--------- | :---------- |
+| **Hard At Start Time**    | Boolean | True | ✗ | Flag that forces the hard constraints to reinitialize at start time. This attribute has effect only if preroll start time is lower than start time. |
+| **Shape Preservation At Start Time**    | Boolean | False | ✗ | Flag that forces the shape preservation constraints to reinitialize at start time. This attribute has effect only if preroll start time is lower than start time. |
+
```
[the remark this answers is not in the document]
Comment (carlosmonteagudoinbibo) [reply docs/maya/fat.md]: Sure! They are aligned now

**Review by josevalverdeinbibo — APPROVED** [review]

File: docs/maya/ribbon.md [hunk 2]
```diff
@@ -56,6 +56,13 @@ Follow this steps to create an AndRibbonMuscle deformer:
 
 ### Advanced Settings
 
+#### Initialization Settings
```
Comment (lucasferryinbibo) [inline docs/maya/ribbon.md:59]: Just a quick reminder that we will have to update the screenshots of the simple setup as it is also required for the Debug Fibers AET entry. maybe we can document this and leave it for the very end so we don't have to do work twice

**Review by lucasferryinbibo — APPROVED** [review]
Just one comment about updating the screenshots in other places

**Pull request merged**